clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/DefaultInputDataInfoProvider.java

```diff
@@ -7,7 +7,7 @@
 import static com.linkedin.venice.vpj.VenicePushJobConstants.KEY_FIELD_PROP;
 import static com.linkedin.venice.vpj.VenicePushJobConstants.MINIMUM_NUMBER_OF_SAMPLES_REQUIRED_TO_BUILD_ZSTD_DICTIONARY;
 import static com.linkedin.venice.vpj.VenicePushJobConstants.PATH_FILTER;
-import static com.linkedin.venice.vpj.VenicePushJobConstants.TIMESTAMP_FIELD_PROP;
+import static com.linkedin.venice.vpj.VenicePushJobConstants.RMD_FIELD_PROP;
```

Comment:
So.... I'm not sure about this.  Timestamp as a long while not as useful as RMD was at least an abstraction that didn't leak the implementation of the full RMD to users.  I can understand wanting to add support for passing the RMD as an 'internal' feature, but I'm not certain I'm sold on it completely superceding the usage of being able to provide a timestamp.  That is unless folks are willing to just drop usage of the timestamp all together and we have some other means to abstract the RMD implementation from them.